### PR TITLE
Passing down PR labels to codeql analysis too

### PR DESCRIPTION
**Pull request by amoghrajesh**
We need to pass down the env variable for PR labels to the codeql analysis running per PR too. 
Failure example: https://github.com/apache/airflow/actions/runs/12065619780/job/33644782061?pr=43774

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by Lee-W — APPROVED** [review]

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -50,6 +50,7 @@ jobs:
       - name: Selective checks
         id: selective-checks
         env:
+          PR_LABELS: "${{ steps.source-run-info.outputs.pr-labels }}"
```
Comment (gopidesupavan) [inline .github/workflows/codeql-analysis.yml:53]: I think this step `source-run-info` not exists in this workflow , so it becomes empty always

Comment (amoghrajesh) [reply .github/workflows/codeql-analysis.yml:53]: Hmm ok. In that case, we skip the label check for codeql? I am not so sure how to do it

Comment (gopidesupavan) [reply .github/workflows/codeql-analysis.yml:53]: Guess lets add this step before that. similar to this ? https://github.com/apache/airflow/blob/main/.github/workflows/ci.yml#L160

```
      - name: "Get information about the Workflow"
        id: source-run-info
        run: breeze ci get-workflow-info 2>> ${GITHUB_OUTPUT}
```

Comment (gopidesupavan) [reply .github/workflows/codeql-analysis.yml:53]: That steps pulls out all the labels and pr information. eg: https://github.com/apache/airflow/actions/runs/12065619879/job/33644790403?pr=43774#step:7:905

**Pull request merged**